airflow/config_templates/config.yml

```diff
@@ -62,10 +62,9 @@
       default: "SequentialExecutor"
     - name: parallelism
       description: |
-        This defines the maximum number of task instances that can run concurrently per scheduler in
-        Airflow, regardless of the worker count. Generally this value, multiplied by the number of
-        schedulers in your cluster, is the maximum number of task instances with the running
-        state in the metadata database.
+        This defines the maximum number of task instances that can run concurrently in
+        Airflow, regardless of the worker or scheduler count. This value represents the maximum number
+        of tasks that can be in the running or queued states in the metadata database.
```

Comment:
this basicly reverts https://github.com/apache/airflow/pull/23768

Can you please share some information to backup the suggested description?

Reply:
Apologies -- my Airflow deployment only had one scheduler, but I thought it had two.

However, this doesn't make a lot of sense to me. Why would the number of schedulers influence a parameter that's used to limit the amount of work that can be done in parallel?

Reply:
Because this value is not synchronized between schedulers due to performance reasons.. Each scheduler only monitors its own tasks, so everyone has their own quota.